examples/webgl_materials_envmaps_hdr.html

```diff
@@ -146,7 +144,7 @@
 				};
 
 				var hdrUrls = genCubeUrls( "../examples/textures/cube/pisaHDR/", ".hdr" );
-				new THREE.HDRCubeTextureLoader().load( hdrType, hdrUrls, function ( hdrCubeMap ) {
+				new THREE.HDRCubeTextureLoader().load( THREE.UnsignedByteType, hdrUrls, function ( hdrCubeMap ) {
```

Comment:
If you use half or float, it unpacks the *.hdr maps into float or half buffers as appropriate.  If you specify UnsignedByte, it keeps the original RGBE byte encoding and does live decoding within the fragment shader (because texture.encoding = THREE.RGBEEncoding.)  RGBE is 8bit per channel, but the E is the exponent.  Thus it is 16 bit floating point with a shared 8 exponent in the former:

r = R * 2^ ( E * 255 - 128.0 )
g = R * 2^ ( E * 255 - 128.0 )
b = R * 2^ ( E * 255 - 128.0 )